tests/Behat/ContentLockContext.php

```diff
@@ -42,6 +38,58 @@ public function createMyNode($type_name, $title): void {
     $this->getSession()->visit($this->locatePath('/node/' . $saved->nid));
   }
 
+  /**
+   * Navigates to a particular node..
+   *
+   * @param string $type_name
+   *   The content type of the node.
+   * @param string $title
+   *   The title of the node.
+   *
+   * @throws \Exception
+   *   Thrown when node is not found.
+   *
+   * @Then I visit the :type_name content titled :title
```

Comment:
How about this? Then I visit the :type_name content with title: :title ?

Reply:
I wanted to keep it consistent with the other steps provided by the context, but I don't oppose this :)